src/survey.cpp

```diff
@@ -338,12 +338,7 @@ void SurveyGrfs(nlohmann::json &survey)
 		if ((c->palette & GRFP_BLT_MASK) == GRFP_BLT_32BPP) grf["blitter"] = "32bpp";
 
 		grf["is_static"] = HasBit(c->flags, GCF_STATIC);
-
-		std::vector<uint32_t> parameters;
-		for (int i = 0; i < c->num_params; i++) {
-			parameters.push_back(c->param[i]);
-		}
-		grf["parameters"] = parameters;
+		grf["parameters"] = span<const uint32_t>(c->param.data(), c->num_params);
```

Comment:
Most of the places the namespaces also there, so should not this be:
```suggestion
		grf["parameters"] = std::span<const uint32_t>(c->param.data(), c->num_params);
```
?

Reply:
span is a type in `src/core/span_type.hpp` it's not in the std namespace.

Reply:
To add to that: `std::span` is only C++20, and we still run C++17 :(